package.json

```diff
@@ -217,8 +218,8 @@
     "react-datetime": "^3.0.0",
     "react-dev-utils": "^8.0.0",
     "react-div-100vh": "^0.5.6",
-    "react-dnd": "^7.4.5",
-    "react-dnd-html5-backend": "^7.4.4",
+    "react-dnd": "^15.1.2",
```

Comment:
Yay! I love updating dependencies ! and I hear that this got us some good stuff... Looking forward to actually playing with the feature.